pkgs/servers/sql/postgresql/pgjwt/default.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchgit}:
+stdenv.mkDerivation rec {
+  name="pgjwt-${version}";
+  version="0.0.1";
+  src = fetchgit {
```

Comment:
Does this rely on submodules? If not, consider using a source archive, e.g., via `fetchFromGitHub`.